### PR TITLE
catppuccin-fcitx5: init at 2022-10-05

### PR DESCRIPTION
**Pull request by pluiedev**
## Description of changes

Packages containing [the Catppuccin themes for Fcitx5](https://github.com/catppuccin/fcitx5/).

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by jopejoe1 — CHANGES_REQUESTED** [review]
Please move this over to the new packages by-name sheme at https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

**Comment by pluiedev** [comment]
Didn't have a great enough internet connection to download like thousands of commit at once without timing out until today... ready for review :+1:

**Review by jopejoe1 — APPROVED** [review]
Result of `nixpkgs-review pr 278312` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>catppuccin-fcitx5</li>
  </ul>
</details>

**Comment by Aleksanaa** [comment]
There was a pr here: https://github.com/NixOS/nixpkgs/pull/267341

**Comment by pluiedev** [comment]
> There was a pr here: #267341

It looks like nothing ever happened to it, though xD

**Comment by Aleksanaa** [comment]
> It looks like nothing ever happened to it, though xD

Yes, but you can ask the original author if he is willing to move forward with the original PR. After all, you don’t want to be left aside for a few months and then “closing in favor of xxx”.

**Comment by jopejoe1** [comment]
Didn't see that one as i was reviewing this one will also give my review to that pr.

**Comment by pluiedev** [comment]
The other PR has been closed in favor of this one — ready for merge

File: pkgs/by-name/ca/catppuccin-fcitx5/package.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "catppuccin-fcitx5";
+  version = "2022-10-05";
```
Comment (jopejoe1) [inline pkgs/by-name/ca/catppuccin-fcitx5/package.nix:8]:
```suggestion
  version = "0-unstable-2022-10-05";
```

**Review by jopejoe1 — CHANGES_REQUESTED** [review]
Missed this one as I reviewed it last time, but version should follow the guidelines for unstable versions as it has no official version.

**Review by jopejoe1 — APPROVED** [review]
Looks good now

**Pull request merged**